src/components/TaskGenerator.js

```diff
@@ -2,39 +2,53 @@ import React, { useState, useEffect } from 'react';
 import { useDispatch, useSelector } from 'react-redux';
 import useBreakpoint, { MEDIA_QUERIES, MODE } from '../hooks/useBreakpoint';
 import { toggleCompleted, updateRandomTask } from '../store/tasks/tasks';
-import tasks from '../data/tasks';
+import useFilterTasks from '../hooks/useFilterTasks';
 import SkillRequirementList from './SkillRequirementList';
 import Category from './Category';
 import Difficulty from './Difficulty';
+import TASKS from '../data/tasks';
 
 export default function TaskGenerator() {
     const [activeTask, setActiveTask] = useState(undefined);
-    const [allTasksCompleted, setAllTasksCompleted] = useState(false);
+    const [noPossibleTasks, setNoPossibleTasks] = useState(false);
+    const isLgViewport = useBreakpoint(MEDIA_QUERIES.LG, MODE.GREATER);
```

Comment:
minor: I know this line was just moved and wasn't added in this PR, but just noticed this is either named incorrectly or working incorrectly 😅 With `MODE.GREATER` this would actually only return true for XL or larger viewports, not LG